default_conf.json.in

```diff
@@ -34,7 +34,8 @@
         "level": "debug",
         "name": "ChronoVisor",
         "filesize": 102400,
-        "filenum": 3
+        "filenum": 3,
+        "flushlevel": ""
```

Comment:
Why do we have empty level as default? Do you want to the code to fall into the last branch in parseFlushLevelConf() to get it set to warn? then why don't we just set it to warn in the default configuration?

Reply:
Makes sense